app/helpers/admin/enterprises_helper.rb

```diff
@@ -9,5 +9,9 @@ def add_check_if_single(count)
         {}
       end
     end
+
+    def select_only_item(size)
+      size == 1 ? 1 : nil
```

Comment:
I think this needs to return the `id` of the producer, rather than just `1`, right?

Reply:
Yeah you are right, my mistake 😞 I will update after the decision made in the previous comment